app/jobs/calendar_importer/events/base.rb

```diff
@@ -52,10 +52,13 @@ def html_sanitize(input)
         # if we get HTML then remove all the attributes from all of
         # the tags so it doesn't interfere with the kramdown step
         doc.css('*').each do |tag|
+          # rubocop:disable Style/HashEachMethods
```

Comment:
why not just to what rubocop asks instead? :)

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashEachMethods

Reply:
Because it is assuming `tag` is an enumerable hash, which it is not, so invoking `each_key` raises a `NoMethodError`. It's just guessing based on text parsing and heuristics, which in this case is wrong

Reply:
fair, will leave this to F to sign off